Tella/Components/TextfieldView.swift

```diff
@@ -5,7 +5,15 @@
 
 import SwiftUI
 import Combine
-
+enum FieldType {
+    case url
+    case username
+    case text
+    case password
+    case code
+    case uwaziUsername
```

Comment:
I think we have already text field type that can be used for  uwaziUsername and  uwaziPassword